src/gs-page.c

```diff
@@ -49,6 +50,7 @@ typedef struct {
 	GtkWidget	*button_install;
 	GsPluginAction	 action;
 	GsShellInteraction interaction;
+	GSettings	*settings;
```

Comment:
The settings doesn't really belong to the operation, and thus the helper. So please add it rather to the private struct of the `GsPage`.